src/pages/sessions/running.js

```diff
@@ -229,55 +230,85 @@ const Running = ({ intl, shortname }) => (
-                        isPublic={settings.isFeedbackChannelPublic}
-                        subscribeToMore={() => {
-                          subscribeToMore({
-                            document: FeedbackAddedSubscription,
-                            updateQuery: (prev, { subscriptionData }) => {
-                              if (!subscriptionData.data) return prev
-                              return {
-                                ...prev,
-                                runningSession: {
-                                  ...prev.runningSession,
-                                  feedbacks: [...prev.runningSession.feedbacks, subscriptionData.data.feedbackAdded],
+                      <Mutation mutation={UpdateFeedbackMutation}>
+                        {updateFeedback => (
+                          <FeedbackChannel
+                            feedbacks={feedbacks}
+                            handleActiveToggle={() => {
+                              updateSettings({
+                                refetchQueries: [{ query: RunningSessionQuery }],
+                                variables: {
+                                  sessionId: id,
+                                  settings: {
+                                    isFeedbackChannelActive: !settings.isFeedbackChannelActive,
+                                  },
                                 },
-                              }
-                            },
-                            variables: { sessionId: id },
-                          })
-                        }}
-                      />
+                              })
+                            }}
+                            handleDeleteFeedback={feedbackId => {
+                              deleteFeedback({
+                                variables: { feedbackId, sessionId: id },
+                              })
+                            }}
+                            handlePublicToggle={() => {
+                              updateSettings({
+                                refetchQueries: [{ query: RunningSessionQuery }],
+                                variables: {
+                                  sessionId: id,
+                                  settings: {
+                                    isFeedbackChannelPublic: !settings.isFeedbackChannelPublic,
+                                  },
+                                },
+                              })
+                            }}
+                            handleUpdateFeedback={(newTags, feedbackId) => {
+                              updateFeedback({
+                                refetchQueries: [{ query: RunningSessionQuery }],
+                                variables: {
+                                  feedbackId,
+                                  sessionId: id,
+                                  tags: newTags,
+                                  settings: {
```

Comment:
Why do we need to pass these settings? I think these should not change if we only update a single feedback?